src/services/consensus-api/service.ts

```diff
@@ -52,6 +58,15 @@ export const makeConsensusApi = (
     return data
   }
 
+  const spec = async () => {
+    const res = await request(`${normalizedUrl}/eth/v1/config/spec`, {
```

Comment:
Check all nodes returns CAPELLA_FORK_VERSION inside this api
It's not documented in https://ethereum.github.io/beacon-APIs/#/Config/getSpec, so this field is not required in API.

Reply:
This could be a problem, thanks!
so I put this logic to a constant and wrote a script to get these variables for each network.